### DISCUSSION TITLE
Building in release mode using cmake fails, using configure method succeeds

### DISCUSSION THREAD

**Discussion by nsarzyns** [OpenDDS/OpenDDS#4529] [Q&A]
I don't know if I'm just being dumb here but following the build instructions for cmake, successfully creates the sln, but if I change the build type to 'Release' it fails because for some reason the linker is still trying to use debug libraries in release mode. For example, the linker for the 'inspect' sub-project tried to link libs like ACEsd.lib, TAOsd.lib...basically all the libs have 'd' appended. I've even tried forcing the CMAKE_CONFIGURATION_TYPES to Release in the root cmakelists without success.

On the other hand, building using the configure file, works perfectly fine. The same sub project links the correct libs for the selected configuration of release.

Both instances are using ACE/TAO 7.1.3 and building on windows.

**Comment by iguessthislldo** [comment]
I recreated the problem from a clean build, which is unfortunate because I'm certain this was working before. I've created issue #4531 from this.
